spec/javascripts/app/pages/contacts_spec.js

```diff
@@ -297,7 +297,7 @@ describe("app.pages.Contacts", function(){
       expect(app.views.ConversationsForm.prototype.initialize).toHaveBeenCalled();
 
       var prefill = app.views.ConversationsForm.prototype.initialize.calls.mostRecent().args[0].prefill;
-      var contacts = app.contacts.filter(function(contact) { return contact.inAspect(app.aspect.get("id")); });
+      var contacts = app.contacts.filter(function(contact) { return contact.person.get("relationship") === "mutual" && contact.inAspect(app.aspect.get("id")); });
```

Comment:
I don't know why @diaspora-code-review didn't complain about this (looks like it is broken, maybe @denschub can have a look at that?), but this line is longer than 120 chars. But you can simply write it in multiple lines, the same as it is in `contacts.js`.

By the way, you can run these styleguide checks locally with `bin/pronto run -c develop`, see here for more information about that: https://wiki.diasporafoundation.org/Styleguide#Automatic_local_review

Reply:
Done